core/backend/src/test/standalone/DeleteDefinitionElements.test.ts

```diff
@@ -0,0 +1,115 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { assert } from "chai";
+import * as path from "path";
+import { Id64, Id64Set, Logger, LogLevel } from "@bentley/bentleyjs-core";
+import { IModel } from "@bentley/imodeljs-common";
+import {
+  CategorySelector, DisplayStyle3d, DrawingCategory, IModelJsFs, InformationPartitionElement, ModelSelector, OrthographicViewDefinition, SnapshotDb, SpatialCategory, Subject,
+} from "../../imodeljs-backend";
+import { IModelTestUtils } from "../IModelTestUtils";
+import { IModelTransformerUtils } from "../IModelTransformerUtils";
+import { KnownTestLocations } from "../KnownTestLocations";
+
+describe("DeleteDefinitionElements", () => {
+  const outputDir: string = path.join(KnownTestLocations.outputDir, "DeleteDefinitionElements");
+
+  before(async () => {
+    if (!IModelJsFs.existsSync(KnownTestLocations.outputDir)) {
+      IModelJsFs.mkdirSync(KnownTestLocations.outputDir);
+    }
+    if (!IModelJsFs.existsSync(outputDir)) {
+      IModelJsFs.mkdirSync(outputDir);
+    }
+    // initialize logging
+    if (false) {
+      Logger.initializeToConsole();
+      Logger.setLevelDefault(LogLevel.Error);
+    }
+  });
+
+  it.only("should delete if not used", async () => {

```

Comment:
Remove `.only`.

Reply:
Removed .only since I am ready to publish this PR.